CDP4Dal/CDPMessageBus.cs

```diff
@@ -68,13 +68,40 @@ public static CDPMessageBus Current
         /// Total number of Calls that have been made to the Listen method
         /// during the lifetime of this instance of <see cref="CDPMessageBus"/>.
         /// </summary>
-        public int ListenerCallCount { get; private set; } 
+        public int ListenerCallCount { get; private set; }

```

Comment:
i see this being introduced but the value does not seem to be set? please remove if not being used

Reply:
It was there, used inside Listen method(I don't know why the viewer/differ marked this as added):
```
        public IObservable<T> Listen<T>(object target = null, string contract = null)
        {
            this.ListenerCallCount++;
            return this.GetOrAddObservable<T>(new EventTypeTarget(typeof(T), target));
        }
```